README.md

```diff
@@ -1 +1,14 @@
-# skyux-spa-addin-entry-form-demo
\ No newline at end of file
+# skyux-spa-addin-entry-form-demo
+
+This SPA demonstrates a SKY Add-in implementation using SKY UX. The example uses the Razor's Edge NXT Gift Entry form and side loads a vertical add-in tab in the same modal dialog tabset.
```

Comment:
Only thing I noticed is that Raiser's Edge is misspelled.

Reply:
Fixed the spelling error. Thanks!